contributors/devel/conformance-tests.md

```diff
@@ -62,6 +62,9 @@ Conformance tests can be run against clusters that have not been created with
 credentials.
 
 ```sh
+# build test binaries and kubectl first by run:
```

Comment:
nit: maybe drop the " by run" or s/run/running/ ? I'd probably say just "build test binaries and kubectl:"

Reply:
fixed, thanks